app/Domain/Goalcanvas/Controllers/BigRock.php

```diff
@@ -68,7 +68,7 @@ public function get($params): Response
                 $bigrock = array("id"=>'', "title" => "", "prpojectId" => "", "author" => '');
             }
 
-
+            dd($bigrock);
```

Comment:
Please remove

Reply:
Why did you resolve the comment but not remove the line?

Reply:
i resolved it before pushing the code, I resolved it locally